### PR TITLE
shared/storage/certpool: implement Bundler and fix x509utils.Names()

### PR DESCRIPTION
**Pull request by amery**

### CODE REVIEW

**Comment by amery** [comment]
Injected a commit on top removing shared/x.MapContains

File: shared/storage/certpool/bundler.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+// Bundler uses two CertPoolers to bundler keys and certificates
+type Bundler struct {
+	Roots x509utils.CertPooler
+	Inter x509utils.CertPooler
+}
+
+// Bundle bundles a key and a certificate into a *tls.Certificate
+func (b *Bundler) Bundle(cert *x509.Certificate, key x509utils.PrivateKey) (
+	*tls.Certificate, error) {
```
Comment (karasz) [inline shared/storage/certpool/bundler.go:24]: I know it is valid, but please do not break lines like this. It is very annoying to me.

**Review by karasz — APPROVED** [review]
GTG

**Comment by amery** [comment]
`shared/v0.1.5` tagged

**Pull request merged**